editor/src/components/canvas/controls/grid-controls-for-strategies.tsx

```diff
@@ -90,13 +96,17 @@ export type GridMeasurementHelperData = {
   rowGap: number | null
   columnGap: number | null
   padding: Sides
+  element: HTMLElement
```

Comment:
Passing around and holding onto DOM elements makes me a little twitchy. It can shift around underneath us as React updates it and we're holding onto a reference to something which could hold onto a lot of memory potentially.